crates/adapter/src/prover/stone_prover.rs

```diff
@@ -34,13 +35,15 @@ const SERIALIZED_PROOF_PATH: &str = "bootloader_serialized_proof.json";
 
 pub struct StoneProver {
     pub cairo_pie: Vec<PathBuf>,
-    pub layout: LayoutName,
+    pub layout: String,
```

Comment:
Why changing from LayoutName to String?

Reply:
I dont want to import LayoutName to the worker,  so i convert to string and parse to layoutname